test/instances/ezETH/EzETHHyperdrive.t.sol

```diff
@@ -54,187 +52,78 @@ contract EzETHHyperdriveTest is HyperdriveTest {
     // mocking.  To test with their deployed code we use a shorter position
     // duration.
     uint256 internal constant POSITION_DURATION_2_WEEKS = 15 days;
```

Comment:
Nit: I know you didn't write this, but we may as well rename this to `POSITION_DURATION_15_DAYS`.